src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/ServiceReflector.cs

```diff
@@ -18,7 +18,7 @@ internal static Type ResolveTypeFromName(string name)
                 }
             }
 
-            return default;
+            return Type.GetType(name, true);
         }
```

Comment:
Could you add a comment at the top along the lines of checking if type is in an already loaded assembly first and then before this line add a comment about the type string needing to be an AssemblyQualifiedName to make CLR load the assembly.